index.html

```diff
@@ -101,5 +106,6 @@ <h1>믿을 수 있는 <br> 판다마켓 중고거래</h1>
       </div>
     </div>
   </footer>
+  <script src="index.js"></script>
```

Comment:
```suggestion
  <script src="index.js" />
```

내부에 children이 없으면 이렇게 표현해주는것이 조금 더 가독성이 좋을것 같아요!

Reply:
또한 html에서 script태그를 사용할 때, defer 혹은 async 옵션이 있는데 그것에 대해서도 한번 알아보시면 좋을 것 같아요!
[참고](https://ko.javascript.info/script-async-defer)